chainer/training/extension.py

```diff
@@ -28,14 +28,10 @@ class Extension(object):
             ``(1, 'iteration')`` by default.
         priority: Default priority of the extension. It is set to
             ``PRIORITY_READER`` by default.
-        invoke_before_training: Default flag to decide whether this extension
```

Comment:
This is not an argument; this is an attribute. I will add a property that raises an error.